### DISCUSSION TITLE
Feature Request: Export sqlite tables

### DISCUSSION THREAD

**Discussion by totalhack** [duckdb/duckdb#6842] [Ideas]
I'm basically trying to convert an existing sqlite database into a proper duckdb database for testing a variety of database flavors in another project from a common set of data. I know I can install/load sqlite and then use `sqlite_attach`, but that is not permanent and those tables seem to get ignored if I do `export database`. DuckDB is also failing when I try to import my sqlite data from `.sql` files, probably because it doesn't like the datetimes represented as strings in that SQL output. 

Is there some recommended approach to this that I have missed?

**Comment by Mytherin** [comment]
DuckDB supports attaching SQLite databases directly using the ATTACH command - see e.g. here https://github.com/duckdblabs/sqlite_scanner/blob/main/test/lite_storage/attach_simple.test

**Comment by totalhack** [comment]
But that still operates on the underlying SQLite db right? I want to load a
SQLite database into DuckDB (into proper duckdb tables/formatting)
independent of SQLite or the extension after the load. My workaround was to
do this with an intermediate step to CSVs but it would be nice to be able
to skip that.

**Reply by Mytherin** [reply]
You can have both a DuckDB and a SQLite database attached at the same time and transfer data between them using standard SQL syntax (e.g. create table as, insert from query, etc)

**Comment by totalhack** [comment]
Sure, of course. Was hoping there was some way to do this without extra
steps, much like you can suck in data from a csv as an import instead of a
logical attachment/view of an underlying file.
